### PR TITLE
i18n support

### PR DESCRIPTION
**Pull request by rojenzaman**
split branch for i18n support

### CODE REVIEW

File: lang/german.sh [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```
Comment (kt97679) [inline lang/german.sh:1]: Since those files will not be launched by themselves we don't need shebang line at the top.

**Comment by kt97679** [comment]
Thank you @rojenzaman ! One minor final nit and we are good to merge :)!

**Comment by rojenzaman** [comment]
Thanks! Do you think update main README.md?

**Comment by kt97679** [comment]
Sure, we can update root README.md. Do you want to do this as part of this PR or in a different PR?

File: lang/README.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+and run **tetris.sh** as:
+```text
+./tetris.sh -l english
+```
+Tetris script will automatically identify the local variables you added.
+
+
+EXAMPLE `language_you_added.sh` file:
+```bash
+i18n_lines_completed="Lines completed: ";
```
Comment (kt97679) [inline lang/README.md:16]: Sorry, just noticed: terminating semicolon is not needed here.

File: tetris.sh [hunk 3]
```diff
@@ -17,12 +17,73 @@
...
+                exit 1
+            }
+            . $script_dir/lang/$user_lang.sh || {            # load language file
+                echo "Failed to load $script_dir/lang/$user_lang.sh"
+                exit 1
+            }
+        }
+    ;;
+    h )
+        usage;
```
Comment (kt97679) [inline tetris.sh:77]: same note on terminating semicolon

File: tetris.sh [hunk 4]
```diff
@@ -17,12 +17,73 @@
...
+    ;;
+    : )
+        echo -e "Missing option argument for -$OPTARG\n" # if value of -l not given
+        usage;
```
Comment (kt97679) [inline tetris.sh:81]: same note on terminating semicolon

File: tetris.sh [hunk 5]
```diff
@@ -17,12 +17,73 @@
...
+        echo "available languages:"
+        ( echo "english (default)" && echo "$available_languages") | sort
+    fi
+    exit 0
+}
+
+# BEGIN OF LANGUAGE LINES
+# if you want see or add other languages, visit $script_dir/lang/ dir and read $script_dir/lang/README.md
+# default localized values english:
+i18n_lines_completed="Lines completed: ";
```
Comment (kt97679) [inline tetris.sh:47]: same note on terminating semicolon

File: lang/README.md (hunk 2 above)
Comment (rojenzaman) [reply lang/README.md]: different PR is suitable

**Comment by rojenzaman** [comment]
all of them are okay

**Comment by kt97679** [comment]
Awesome, thank you very much for the changes! We are good to merge. The last question we need to agree on is how to change root README.md and if this should be part of this PR?

**Comment by kt97679** [comment]
Ah, sorry, noticed your comment on the README. In this case I'm merging the change. Woohoo :)!

**Comment by rojenzaman** [comment]
It is useful to add language support in root readme.md. other than that i think it is more convenient for you to prepare the readme.md

**Comment by rojenzaman** [comment]
Thanks for merged it!

**Pull request merged**